bedrock/firefox/templates/firefox/whatsnew/includes/mofo-donate-cta.html

```diff
@@ -0,0 +1,36 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% if LANG.startswith('en-') %}
+  {% set donate_cta = "Donate to the Mozilla Foundation" %}
+{% elif LANG == 'fr' %}
```

Comment:
**Nit:** so with the default `else` now, it doesn't have to be here again for all en-* locales specifically?

```suggestion
{% if LANG == 'fr' %}
```